result/src/test/kotlin/com/github/kittinunf/result/ResultTests.kt

```diff
@@ -2,11 +2,9 @@ package com.github.kittinunf.result
 
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.runBlocking
-import org.hamcrest.CoreMatchers.equalTo
-import org.hamcrest.CoreMatchers.instanceOf
-import org.hamcrest.CoreMatchers.notNullValue
-import org.hamcrest.CoreMatchers.nullValue
+import org.hamcrest.CoreMatchers.*
```

Comment:
Ideally we should have this in the git tracking `.idea/`, but would you mind revert this back to non star version?

Reply:
OK